### PR TITLE
Add dev-preview2.json

### PR DESCRIPTION
**Pull request by dprince**

### CODE REVIEW

File: default.json [hunk 1]
```diff
@@ -23,7 +20,8 @@
       "groupName": "openstack-k8s-operators",
       "matchPackagePatterns": ["^github.com/openstack-k8s-operators"],
       // force pseudo versioning
-      "versioning": "regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)-(?<build>\\d{14})-(?<revision>[a-f0-9]{12})$",
+      //"versioning": "regex:^(?<major>\\d+)\\.(?<minor>\\d+)\\.(?<patch>\\d+)-(?<build>\\d{14})-(?<revision>[a-f0-9]{12})$",
+      "versioning": "regex:^(?<major>0)\\.(?<minor>1)\\.(?<patch>\\d+)-(?<build>\\d{14})-(?<revision>[a-f0-9]{12})$",
```
Comment (gibizer) [inline default.json:24]: This probably needs to be ?<minor>3 as default.json is for the main branch

File: dev-preview2.json [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  },
+  "packageRules": [
+    {
+      "matchPackageNames": ["go"],
+      "enabled": false
+    },
+    {
+      "groupName": "openstack-k8s-operators",
+      "matchPackagePatterns": ["^github.com/openstack-k8s-operators"],
+      "allowedVersions": "< 0.3.0",
```
Comment (gibizer) [inline dev-preview2.json:22]: I think it won't pick up pseuodversions just tagged one but over slack you said we want to track pseudoversions so this needs to be adapted

File: default.json (hunk 1 above)
Comment (dprince) [reply default.json]: I don't actually think we need any changes here. (this was in error). Latest update should remove it

File: dev-preview2.json (hunk 2 above)
Comment (dprince) [reply dev-preview2.json]: see my latest update. This was just old

**Pull request closed without merging**